inc/serverstate.class.php

```diff
@@ -167,7 +167,7 @@ public static function add_action_link($input)
 						),
 						admin_url('/')
 					),
-					__('Settings')
+					__( 'Settings', 'serverstate' )
```

Comment:
Small hint, we should sanitize strings from translation files, so we should use `esc_html__` instead of `__`.